routers/web/user/setting/profile.go

```diff
@@ -48,16 +48,13 @@ func Profile(ctx *context.Context) {
 }
 
 // HandleUsernameChange handle username changes from user settings and admin interface
-func HandleUsernameChange(ctx *context.Context, user *user_model.User, newName string) error {
-	// Non-local users are not allowed to change their username.
-	if !user.IsLocal() {
-		ctx.Flash.Error(ctx.Tr("form.username_change_not_local_user"))
-		return fmt.Errorf(ctx.Tr("form.username_change_not_local_user"))
-	}
-
+func HandleUsernameChange(ctx *context.Context, user *user_model.User, newName string, onlyCapitalization bool) error {
 	// rename user
-	if err := user_service.RenameUser(ctx, user, newName); err != nil {
+	if err := user_service.RenameUser(ctx, user, newName, onlyCapitalization); err != nil {
```

Comment:
I guess the function `RenameUser has the ability to check `onlyCapitalization`, why it needs to be passed as an argument?

Reply:
The third parameter removed.